app/src/main/java/de/westnordost/streetcomplete/quests/way_lit/WayLitAnswer.kt

```diff
@@ -0,0 +1,6 @@
+package de.westnordost.streetcomplete.quests.way_lit
+
+sealed class WayLitOrIsStepsAnswer
+object IsActuallyStepsAnswer : WayLitOrIsStepsAnswer()
+
+data class WayLitAnswer(val value: WayLit) : WayLitOrIsStepsAnswer()
```

Comment:
Yeah, it's a shame that an enum cannot be a subclass of a sealed class. However, since Kotlin 1.5, there are also `sealed interface`s. So it should be possible to write

```kotlin
sealed interface WayLitAnswer
object IsActuallyStepsAnswer : WayLitAnswer
enum class WayLit {
   //... etc
}  : WayLitAnswer
```

Reply:
Nice, that avoids much of the overhead that the sealed class produced :)

Done in [`3493bd6` (#3334)](https://github.com/streetcomplete/StreetComplete/pull/3334/commits/3493bd6458d21bb64f0bf111a88c73f34e732666)